projects/plutus/inc/current_calibration.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+// LTC 2484 calibration module
+
+#include "current_sense.h"
+
+#define CALIBRATION_SAMPLES 10
+
+typedef struct {
+  LtcAdcStorage adc_storage;
```

Comment:
Why do you allocate all ADC storage here?

Reply:
Forgot to change that back to a pointer. Fixed